NuGet.config

```
@@ -3,10 +3,10 @@
   <!-- Only specify feed for Arcade SDK (see https://github.com/Microsoft/msbuild/issues/2982) -->
   <packageSources>
     <clear />
-    <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
-    <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
```

CI failure: System.CommandLine.Rendering not found. Turns out it is being published in the [dotnet5 feed](https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet5):

https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet5/NuGet/System.CommandLine.Rendering/overview/2.0.0-beta1.20074.1

I don't yet know if we will need dotnet7 too. The CI might have had a hard stop after encountering the System.CommandLine.Rendering failure and it didn't tell us if there were a package not found that belongs to dotnet7. I guess the next CI run will tell us.

```suggestion
    <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
```